modules/core/test/test_intrin_utils.hpp

```diff
@@ -1573,22 +1573,22 @@ template<typename R> struct TheTest
         return *this;
     }
 
-#if 0
     TheTest & test_loadstore_fp16()
     {
+#if CV_SIMD_FP16
         printf("test_loadstore_fp16 ...\n");
         AlignedData<R> data;
         AlignedData<R> out;
 
         // check if addresses are aligned and unaligned respectively
-        EXPECT_EQ((size_t)0, (size_t)&data.a.d % VTraits<R>::max_nlanes);
-        EXPECT_NE((size_t)0, (size_t)&data.u.d % VTraits<R>::max_nlanes);
-        EXPECT_EQ((size_t)0, (size_t)&out.a.d % VTraits<R>::max_nlanes);
-        EXPECT_NE((size_t)0, (size_t)&out.u.d % VTraits<R>::max_nlanes);
+        EXPECT_EQ((size_t)0, (size_t)&data.a.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_NE((size_t)0, (size_t)&data.u.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_EQ((size_t)0, (size_t)&out.a.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
+        EXPECT_NE((size_t)0, (size_t)&out.u.d % (sizeof(typename VTraits<R>::lane_type) * VTraits<R>::vlanes()));
 
         // check some initialization methods
         R r1 = data.u;
-        R r2 = vx_load_expand((const hfloat*)data.a.d);
+        R r2 = vx_load(data.a.d);
```

Comment:
According to https://github.com/opencv/opencv/pull/25196/files#r1575853772, input data should be of type `v_float16`, so no conversion is needed. Replaced with a simple `vx_load` instead.